src/main/kotlin/org/jitsi/nlj/rtcp/RtcpRrGenerator.kt

```diff
@@ -35,15 +39,15 @@ import org.jitsi.utils.ms
  */
 private data class SenderInfo(
     var lastSrCompactedTimestamp: Long = 0,
-    var lastSrReceivedTime: Long = 0,
+    var lastSrReceivedTime: Instant? = null,
     // The media type doesn't affect RTCP RR/SR generation. Initialize with a dummy value.
     var statsSnapshot: IncomingSsrcStats.Snapshot = IncomingSsrcStats.Snapshot(mediaType = MediaType.VIDEO)
 ) {
-    private fun hasReceivedSr(): Boolean = lastSrReceivedTime != 0L
+    private fun hasReceivedSr(): Boolean = lastSrReceivedTime != null
 
-    fun getDelaySinceLastSr(now: Long): Long {
+    fun getDelaySinceLastSr(now: Instant): Long {
         return if (hasReceivedSr()) {
-            ((now - lastSrReceivedTime) * 65.536).toLong()
+            Duration.between(now, lastSrReceivedTime).times(65536).seconds
```

Comment:
I think this should be `Duration.between(lastSrReceivedTime, now)` and in milliseconds instead of seconds.

Reply:
You're right about the order of the subtraction, but this value is in 1/65536 of a second, so once I've done `.times(65535)` the `seconds` value should be correct.

This is confusing, though, so I'll add a comment.